rollup/main/default/classes/RollupTests.cls

```diff
@@ -2646,7 +2646,7 @@ private class RollupTests {
     Rollup.maxQueryRowOverride = 1;
 
     Test.startTest();
-    Rollup.performFullRecalculation('Amount', 'AccountId', 'Id', 'AnnualRevenue', 'Account', 'Opportunity', 'SUM', null);
+    Rollup.performFullRecalculation('Amount', 'AccountId', 'Id', 'AnnualRevenue', 'Account', 'Opportunity', 'SUM', '');
```

Comment:
Ah, I can revert this - good catch. I had found that Apex has an issue with static methods that have been overloaded where the only difference with them (other than the last argument's type signature) was the visibility level of the method. So before I renamed one of them to `performFullRecalculationInner`, the compiler was complaining that `null` was ambiguous as an argument since there were now two methods that it could go towards, both named `performFullRecalculation`. In the end, I renamed one of them to include `inner` because while the above change made Apex happy, the LWC associated with the `@AuraEnabled` version of the method was somehow still able to "see" the private static method ... rendering the whole change here moot. I just forgot to back it out amidst the other work - great catch!

tl;dr - me monkey, smooth brain forgot